docker/sequencErr/1.0.0/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM registry.access.redhat.com/ubi7/python-36 AS builder
```

Comment:
It seems `seqeuncerr` doesn't build correctly on Debian systems, which is why I'm using Red Hat instead of `conda-base` here. I told Pandurang and he said they've only tested on the cluster. But it works with Red Hat as the base image.